### DISCUSSION TITLE
Google Summer of Code 2022 CNCF Project Applications

### DISCUSSION THREAD

**Discussion by nate-double-u** [cncf/mentoring#469] [Announcements]
Hello!

CNCF has been participating in the Google Summer of Code during the [last few years](https://www.cncf.io/blog/2021/11/01/16-cncf-interns-graduated-from-google-summer-of-code-gsoc-2021/) and now we are ready to start the 2022 program cycle.
 
All CNCF projects are eligible to participate in the Google Summer of Code. Please start working on the project ideas, and submit them via pull request: https://github.com/cncf/mentoring/blob/main/summerofcode/2022.md#project-ideas. 

The deadline to submit the project ideas is February 14, 2022.

[Google Summer of Code 2022 Timeline](https://developers.google.com/open-source/gsoc/timeline)

Thanks!

**Comment by nate-double-u** [comment]
A friendly nudge: the GSoC project application deadline is approaching! Please submit your GSoC'2022 ideas by Monday, Feb 14th.

**Comment by idvoretskyi** [comment]
CNCF has formally applied as an organization this weekend, yay! 

We'll keep this thread updated with fresh news, for now, we can close this issue - https://github.com/cncf/mentoring/issues/465.

**Comment by nate-double-u** [comment]
Hello everyone,

We've been asked to provide some more information about the projects. I've gone through and commented on each project's PR, but thought I'd also post here in case anyone had any general questions.

As per the [Defining a Project Idea list](https://google.github.io/gsocguides/mentor/defining-a-project-ideas-list), could you please update any project suggestions you've submitted with the following (if not already provided):

a) a project title/description 
b) more detailed description of the project (2-5+ sentences) 
c) expected outcomes 
d) skills required/preferred 
e) possible mentors 
f) expected size of the project (175 or 350 hours) 
g) an easy, medium, or hard difficulty rating of each project.

~Please update your projects by March 3, 2022.~ Thanks!

Note, there is a new deadline: please update by **Tues, March 1 at 5pm PT**.

**Comment by nate-double-u** [comment]
Hi everyone,

I've gone through and added template information to all of the projects. Please go through your projects and add any additional required information.

https://github.com/cncf/mentoring/issues/523

Note there is a new deadline: Please make these updates by **Tues, March 1 at 5pm PT**.

Thanks!

**Comment by nate-double-u** [comment]
We got all the projects updated. Thanks for the effort on short notice everyone!

**Comment by RainbowMango** [comment]
Hi @nate-double-u ,

According to the timeline, `March 7 - 18:00 UTC` should be the time to list accepted programs.
Any update?

**Comment by nate-double-u** [comment]
Hi Everyone, I'm a little late to say it, but CNCF has been selected as a Google Summer of Code 2022 mentor organization! 🎉

You can find the the full list of accepted 2022 GSoC mentoring organizations at https://g.co/gsoc.

**Comment by nate-double-u** [comment]
Hi again everyone! I've sent invitations [for the CNCF org](https://summerofcode.withgoogle.com/organizations/cncf) to all the **mentors** that I have an email address for. 

**If you are a mentor** and have not received one yet, I may not have your email address. Could you please provide it on this form:

https://forms.gle/339NecJLfXPuDahh8

**Reply by daemon1024** [reply]
Hey @nate-double-u 👋🏽,
I am unable to access the form as well.
![image](https://user-images.githubusercontent.com/47106543/158400184-824155b4-4d82-479e-81c6-fec52f3ffbba.png)

**Reply by nate-double-u** [reply]
Hi, thanks for letting me know. I've just made an update to the settings. Sorry about that, could you try again?

**Reply by daemon1024** [reply]
I can access it now 🚀 Thanks a lot 😄

**Reply by nate-double-u** [reply]
Hi Everyone,

I still don't have email addresses for all of the **mentors**, so I'm not able to invite you to [the CNCF org](https://summerofcode.withgoogle.com/organizations/cncf).

**If you are a mentor listed below** and have not received an invitation to the GSoC CNCF org yet, I don't have your email address. Could you please provide it on this form:

https://forms.gle/339NecJLfXPuDahh8

@varshaprasad96
@cpanato
@sedefsavas
@richcase
@FogDong
@Somefive
@Hexilee
@zhangyangyu
@shiyuhang0
@zhongzc
@breeswish
@aimichelle
@oazizi000
@nserrino
@Shelley-BaoYue
@jaypume
@MooreZheng
@JimmyYang20
@squat
@onprem
@matej-g
@znewman01
@THS-on
@Jonsy13
@rajdas98
@Adarshkumar14
@warunicorn19

Thanks!

**Reply by warunicorn19** [reply]
Filled in the form, Thanks @nate-double-u

**Reply by matej-g** [reply]
Sorry about that, I have filled the form now, thanks @nate-double-u.

**Reply by varshaprasad96** [reply]
Just filled in the form, thanks @nate-double-u

**Reply by warunicorn19** [reply]
hey @nate-double-u, I just received an invite, but the link doesn't seem to be working. Is there something wrong with the GSOC'22 website?

**Reply by nate-double-u** [reply]
> hey @nate-double-u, I just received an invite, but the link doesn't seem to be working. Is there something wrong with the GSOC'22 website?

Not that I'm aware of. 

Note, I did have to turn off "prevent cross-site tracking" in my browser to sign up correctly when I did it.

**Reply by warunicorn19** [reply]
I tried that too, doesn't seem to be working for me, sent an email to gsoc-support.

**Reply by aimichelle** [reply]
Hi @nate-double-u , I'm one of the mentors on one of the Pixie projects. I submitted my information on the form a couple of weeks back, but haven't received an invite to the platform yet. Could you take a look? My email is michellenguyen@pixielabs.ai. Thanks!

**Reply by nate-double-u** [reply]
Hi @aimichelle, I've just resent the invite! Please let me know if you've not received it.

**Reply by nate-double-u** [reply]
Hi Everyone,

I still don't have email addresses for all of the **mentors**, so I'm not able to invite you to [the CNCF org](https://summerofcode.withgoogle.com/organizations/cncf).

**If you are a mentor listed below** and have not received an invitation to the GSoC CNCF org yet, I don't have your email address. Could you please provide it on this form:

https://forms.gle/339NecJLfXPuDahh8

@rajdas98 
@JimmyYang20
@jaypume
@Shelley-BaoYue
@breeswish
@richcase


If there's been an error and you have provided your email address, let me know and we'll see about sorting it out. Thanks!

**Reply by Hexilee** [reply]
Hi @nate-double-u , I'm one of the mentors of Chaos Mesh project. I submitted the form weeks but also haven't received an invitation. Could you please resent one to lichenxi@pingcap.com? Thx.

**Comment by Mostafa-wael** [comment]
Hello everyone, I am really interested in a lot of projects in this [list](https://github.com/cncf/mentoring/blob/main/summerofcode/2022.md) and want to apply for them as a GSOC student this year. Any ideas about what should I do? how to contact the mentor? etc..

**Reply by nate-double-u** [reply]
Hi @Mostafa-wael,

Take a look at this discussion:
* https://github.com/cncf/mentoring/discussions/565

There is a list of project related discussions where you can reach out to the mentors and talk about your ideas.

**Reply by Mostafa-wael** [reply]
Thanks nate! I will check it

**Comment by 0904-mansi** [comment]
Hey Everyone! Where can we find the template for writing a GSoC proposal for contributors?

**Reply by nate-double-u** [reply]
I don't think there are any templates for proposals to the CNCF orgs, but the [GSoC Contributor/Student Guide](https://google.github.io/gsocguides/student/) has a section on [writing proposals](https://google.github.io/gsocguides/student/writing-a-proposal).
